recipes/pysm3/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pysm3" %}
+{% set version = "3.1.dev238" %}
```

Comment:
We generally don't release dev packages. Is there a full release that you can target?

Reply:
It will be ready in a few weeks. But in the meantime it would be nice to have the infrastructure ready. I won't be releasing any dev release after this.

Reply:
OK, I think this will have to wait then.  There are a couple of issues related to releasing dev packages

Reply:
@scopatz ok, then I packaged 3.1.0 now.